app/models/host/managed_extensions.rb

```
@@ -20,6 +22,7 @@ def setReboot
     else
       logger.info "ForemanDiscovery: reboot result: failed"
     end
+    true
```

Should we return the failure case to the UI so the users knows to go manually reboot the host?